CHANGES.rst

```diff
@@ -2,6 +2,15 @@
 Changes for Crate Admin Interface
 =================================
 
+Unreleased
+==========
+
+Fixes
+-----
+
+- Fixed an issue where a nested column with a space in it would not be as one,
```

Comment:
The "not be as one" part may be a bit confusing without more context.

Reply:
Aha, im missing the word "identified" :see_no_evil: